README.md

```diff
@@ -126,67 +132,21 @@ We were able to run the training for 100 epochs in 25 minutes on 2 P2.8x EC2 ins
-We were able train this model in an hour on 2 P2.8x EC2 instances running 2 process status processes and 16 worker processes for 200000 steps and reduce the loss to 0.82 averaged across the 16 workers.
-
-Running the evaluation script on the trained model achieves an accuracy of 77%:
-
-    python $EFS_MOUNT/deeplearning-cfn/examples/tensorflow/models/tutorials/image/cifar10/cifar10_eval.py \
-    --data_dir=$EFS_MOUNT/cifar10_data/ \
-    --eval_dir=$EFS_MOUNT/deeplearning-cfn/examples/tensorflow/eval \
-    --checkpoint_dir=$EFS_MOUNT/deeplearning-cfn/examples/tensorflow/train
-
-You can visualize the training on [TensorBoard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) by running the following command on the master node. TensorBoard starts running on the private IP address of the master instance and port 6006. Make a note of this IP address because you will use it in the next command.
-
-    tensorboard --logdir $EFS_MOUNT/deeplearning-cfn/examples/tensorflow/train
+### Running Distributed Training on TensorFlow with Horovod
+Horovod is a distributed training framework to make distributed deep learning easy and fast. You can get more information about the advantages of using [Horovod](https://github.com/uber/horovod) to do distributed training with Tensorflow or other framework.
+Starting from DLAMI Conda v19.0, it comes with example pre-configured scripts to show how you can train model with Tensorflow and Horovod on multi gpus/machines.
 
-Now use SSH port forwarding to see TensorBoard on your local computer. Run a command similar to the following on the local computer. (This uses SSH agent forwarding for credentials.)
+The following example shows how to train a ResNet-50 model with synthetic data.
 
-    #In this example, 192.0.2.0 is the private IP of the master and 203.0.113.0 is the public ip of the master instance, ec2-user is the userid of the master if Instance Type is Amazon Linux
-    ssh -l ec2-user -L 6006:192.0.2.0:6006 203.0.113.0
+    cd ~/examples/horovod/tensorflow
+    vi hosts
 
-To see TensorBoard, open [http://localhost:6006](http://localhost:6006) on a browser. For more information, see [TensorBoard: Visualizing Learning](https://www.tensorflow.org/get_started/summaries_and_tensorboard).
+You should be able to see `localhost slots=8` in your hosts file. The number of slots means how many GPUs you want to use in that machine to train your model. Also, you should append your worker nodes to the hosts file, and assign GPU number to it. To know how many GPUs available in your instance, run `nvidia-smi`. After the change, your hosts file should look like
+```
+localhost slots=<#GPUs>
+<worker node private ip>=<#GPUs>
+......
+```
+Also, you can simply calculate how many GPUs totally you will use to train the model. The total number equal to the sum of slots in each machine. For example, you have 1 master node and 2 worker nodes, and you want to use 8 GPUs in each of them, the total number of GPUs = 24.
```

Comment:
Slightly better structure for the first sentence in para:
Also, you can simply calculate how many GPUs totally you will use to train the model =>
You can easily calculate the number of GPUs you'll use to train the model by summing up the slots available on each machine. Note that: the argument passed to the train_synthetic.sh script below is passed to -np parameter of mpirun. The -np argument represents the total number of processes and the slots argument in hostfile represents the split of those processes per machine.

Reply:
@nskool done.